tests/io/test.xpi.js

```diff
@@ -542,19 +532,17 @@ describe('Xpi.getFileAsStream()', function() {
         expect(err.message).toContain('¡hola!');
       });
   });
-
 });
 
-describe('Xpi.getFilesByExt()', function() {
-
+describe('Xpi.getFilesByExt()', function getFilesByExtCallback() {
```

Comment:
@pdehaan If we end up concluding we don't need the rule I'd rather disable it centrally in the shared config since we're trying to make our config more consistent across repos. Of course there might be some cases where we _do_ need overrides but we'd probably want to keep that to an absolute minimum.